andela/ah-centauri-frontend
# feature(reset password): users should be able to reset their passwords

#### What does this PR do?
A user should be able to reset his/her password when he/she forgets it, or for any other reason

#### Description of Tasks to be completed?
- create component where users can submit their email address and receive a reset password link
- create component where users can submit their new passwords and hence successfully reset their passwords
- write tests for said feature
- The user should receive **descriptive** validation errors in case of any.

#### How should this be manually tested?
- Clone the project: git clone https://github.com/andela/ah-centauri-frontend
- cd into the project folder
- checkout to branch `feature/164829161-password-reset `
- Install dependencies: yarn install
- Launch development server: yarn start
- enter `/reset` after url to access the reset password page
- enter your email address
- go to your inbox and click on the link provided which will redirect you
- enter new password, enter the password again to confirm they are a match

#### Any background context you want to provide?
N/A

#### What are the relevant Pivotal Tracker Stories?
[#164829161](https://www.pivotaltracker.com/story/show/164829161)

#### Screenshots
- get reset password link
![image](https://user-images.githubusercontent.com/40663828/57305761-38da4e80-70ea-11e9-80ca-47ea5c10582f.png)

- reset password form
![image](https://user-images.githubusercontent.com/40663828/57305778-41cb2000-70ea-11e9-915a-99fd26b602b7.png)
